hw03_NatsarS.py

```diff
@@ -0,0 +1,68 @@
+#################################################################################
+# Author: Salaam Natsar
+# Username: NatsarS
+#
+# Assignment: Homework 03
+# Purpose: Created a sun flower
+# Google Doc Link: https://docs.google.com/document/d/1gRRxdAQvTMhQ6pwa0BwnBMu09rZ2cglwKLnd5j2zvpg/edit?usp=sharing
+#################################################################################
+# Acknowledgements:
+# https://stackoverflow.com/questions/32804572/how-to-hide-a-turtle-icon-pointer-in-python
+#
+#################################################################################
+import turtle
+
+def main():
```

Comment:
You don't need to fix this one if you don't want to, but `main` is typically the last function defined in the program, not the first.

Reply:
Resolve this conversation after you've read it, regardless if you fix it or not.